packages/storybook/stories/va-search.stories.jsx

```diff
@@ -54,3 +52,34 @@ WithButtonText.args = {
   ...defaultArgs,
   'button-text': 'Search',
 };
+
+const SuggestionsTemplate = ({
+  'button-text': buttonText,
+  'value': value,
+  label,
+  suggestions,
+}) => (
+  <div style={{ height: '250px' }}>
+    <VaSearch
+      buttonText={buttonText}
+      value={value}
+      label={label}
+      onInput={e => console.log(e)}
+      onSubmit={e => console.log(e)}
+      suggestions={suggestions}
+    />
+  </div>
+);
```

Comment:
I wanted to see if I could make this be more like a real app and I'm not sure how an engineer is supposed to get the value in order to update `suggestions`. Using `e.target.value` in `onInput` doesn't seem to reflect the changes from typing. Am I missing something?

Reply:
I'm able to access the value of the input when it fires using `e.nativeEvent.path[0].value` in Storybook.

Reply:
This is a screenshot of the component being used in the Stencil dev server. I'm using `event.composedPath()[0].value` to retrieve the value of the search input field.

<img width="1792" alt="Screen Shot 2022-04-20 at 15 33 04" src="https://user-images.githubusercontent.com/36863582/164308686-9cebf3d4-9cd2-4a12-ba3c-959cf212e112.png">

Reply:
This is what I had in mind:
```jsx
const handleInput = (event) => {
  const inputValue = event.composedPath()[0].value;
// const inputValue = event.nativeEvent.composedPath()[0].value // React
// we want to start offering suggestions when the user has entered at least 3 characters
  if (inputValue.length > 2) fetchSuggestions();
}

const handleSubmit = (event) => {
  const inputValue = event.detail.value;
  // const inputValue = event.nativeEvent.detail.value; // React
  // prevent submitting if input is empty (optional)
  if (!inputValue) return;
  submitQuery(inputValue);
}
```

where `fetchSuggestions` updates the suggestions array and `submitQuery` navigates the user to the results page or updates results for the current page.

Reply:
I've updated the `With Suggestions` example to showcase how this component works and to aid engineers in implementing their own handlers.